pkg/store/verifiable/store.go

```diff
@@ -43,6 +43,8 @@ type Store interface {
 	GetPresentationIDByName(name string) (string, error)
 	GetCredentials() ([]*Record, error)
 	GetPresentations() ([]*Record, error)
+	RemoveCredential(name string) error
+	RemovePresentation(name string) error
```

Comment:
function's names should be `RemoveCredentialByName` and `RemovePresentationByName`

Reply:
Done